databrowser/src/domain/api/client.ts

```diff
@@ -1,23 +1,22 @@
-import { inject } from '@vue/runtime-core';
-import { AxiosInstance, AxiosResponse } from 'axios';
 import { useQuery } from 'vue-query';
-import { UseQueryReturnType } from 'vue-query/lib/vue/useBaseQuery';
+import { QueryFunction, QueryKey } from 'react-query/types/core';
```

Comment:
Why do we have `react-query` as a dependency?

Reply:
The library `vue-query` relies on the type definitions of `react-query`, because it is a port of `react-query`. I think there are no real react dependencies involved

Reply:
Regarding the pagination bug: well spotted and thank you for the good description to reproduce the error!

I played a little bit around and found out that the pagination starts to work again as soon as the page size changes. Very strange, I'm going to take a look at it.

Reply:
I see 🙈 Nasty shit!